src/archivedb.js

```diff
@@ -272,6 +272,17 @@ class ArchiveDB {
     return results;
   }
 
+  async getTimestampsByURL(url) {
+    const tx = this.db.transaction("resources");
+    const results=[]; 
+    for await (const cursor of tx.store.iterate()) {
```

Comment:
This iterates over all the URLs in the data store, we just want to query the URL that we want.
This is possible to do already with a key range here:
https://github.com/webrecorder/wabac.js/blob/main/src/archivedb.js#L582